### ISSUE TITLE
Application crashes with message exception caused by a factory already set on inflator

### ISSUE THREAD

**Issue by starapple2** [mikepenz/Android-Iconics#359]
I'm using Android Studio 3.1 and in attempting to implement Android Iconics my app crashed on launch. The Logcat reports "java.lang.IllegalStateException: A factory has already been set on this LayoutInflater".

The IDE says "'setFactory(android.view.LayoutInflater, android.support.v4.view.LayoutInflaterFactory)' is deprecated" and struck out setFactory().

What may the fix be?

**Comment by mikepenz** [comment]
seems you already have a `LayoutFactory` added. 

Do you need the iconics support in all default view's or can you use the `IconicsView`'s?

In my opinion the best is anyways still to use the `IconicsDrawable` programatically

**Comment by starapple2** [comment]
Hi Mike, I switched to alternative three and things worked fine. I combed my code for other instances of 
LayoutFactory. I assumed that some other class or function was quietly using it since there's no such literal reference.

Somehow Gradle just seemed to be quirky with the package because after I restarted Android Studio, the mikepenz external classes were listed. Then last night quite of of the blue IconicsContextWrapper turned re with the message unable to resolve symbol. import com.mikepenz.iconics.context.IconicsContextWrapper;" was dimmed with the message that it wasn't used.

When I checked the external classes not one mikepenz was listed. So I decided to fool Gradle by copy and pasting all the implementations in build.gradle, making it ask to be synced. Voila! All the classes returned and the red lines went away. During all of this, there were no build errors.

I'll stick with using the wrapper.

Thanks.
